benchmark/test/test_framework.py.in

```diff
@@ -69,41 +82,15 @@ def sanitize_json(parsed_input, sanitize_all: bool = False):
         return parsed_input
 
 
-def sanitize_json_in_text(lines: List[str]) -> List[str]:
-    """Sanitizes all occurrences of JSON content inside text input.
+def determinize_json_text(input: str) -> List[str]:
```

Comment:
determinize is not a word. Either sanitize or determine

Reply:
it's a word often used to describe the conversion from a non-deterministic automaton to a deterministic one, I thought it might be appropriate. sanitize is a suitable alternative.